src/model_io/urdf/src/URDFModelExport.cpp

```diff
@@ -106,10 +106,120 @@ bool exportInertial(const SpatialInertia &inertia, xmlNodePtr parent_element)
     return ok;
 }
 
+
+
+/*
+ * Add a <visual> or <collision> URDF element to the specified parent element for the specified solid shapes.
+ *
+ * If parent_element contains a pointer to a tag <parent_element></parent_element>, this function modifies it to be something like:
+ * <parent_element>
+ *   <visual>
+ *     <origin xyz="0 0 0" rpy="0 0 0" />
+ *     <geometry>
+ *       <box size="1 1 1" />
+ *     </geometry>
+ *     <material name="Cyan">
+ *       <color rgba="0 1.0 1.0 1.0"/>
+ *     </material>
+ *   </visual>
+ * <parent_element/>
+ * where the actual values of the added element and attributes depend on the input solidShape.
+ */
+enum exportSolidShapeType {
```

Comment:
This name is confusing. since SolidShapeType could imply either box, cylinder, sphere or mesh.

Maybe we can call it `exportPropertyType` or `exportLinkPropertyType`?